apps/bmd/src/AppRoot.tsx

```diff
@@ -207,7 +210,11 @@ const initialAppState: Readonly<State> = {
 // Sets State. All side effects done outside: storage, fetching, etc
 type AppAction =
   | { type: 'processAdminCard'; electionHash: string }
-  | { type: 'processPollWorkerCard'; isPollWorkerCardValid: boolean }
+  | {
+      type: 'processPollWorkerCard'
+      isPollWorkerCardValid: boolean
+      talliesOnCard: Optional<CardTally>
```

Comment:
This would be more idiomatic:
```suggestion
      talliesOnCard?: CardTally
```